varnish-mixin/panels.libsonnet

```diff
@@ -0,0 +1,267 @@
+            signals.cache.cacheHits.asTarget(),
+          ],
+          description='The rate of cache hits.'
+        )
+        + g.panel.stat.standardOptions.withUnit('/ sec')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Cache hit pass stat
+      cacheHitPassPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Cache hit pass',
+          targets=[
+            signals.cache.cacheHitPass.asTarget(),
+          ],
+          description='Rate of cache hits for pass objects (fulfilled requests that are not cached).'
+        )
+        + g.panel.stat.standardOptions.withUnit('/ sec')
+        + g.panel.stat.panelOptions.withTransparent(true),
+
+      // Session queue length stat
+      sessionQueueLengthPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Session queue length',
+          targets=[
+            signals.sessions.sessionQueueLength.asTarget(),
+          ],
+          description='Length of session queue waiting for threads.'
+        )
+        + g.panel.stat.standardOptions.withUnit('none')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Thread pools stat
+      poolsPanel:
+        commonlib.panels.generic.stat.base.new(
+          'Pools',
+          targets=[
+            signals.threads.threadPools.asTarget(),
+          ],
+          description='Number of thread pools.'
+        )
+        + g.panel.stat.standardOptions.withUnit('none')
+        + g.panel.stat.panelOptions.withTransparent(true)
+        + g.panel.stat.options.withGraphMode('none'),
+
+      // Backend connections timeseries
+      backendConnectionsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Backend connections',
+          targets=[
+            signals.backend.backendConnectionsAccepted.asTarget(),
+            signals.backend.backendConnectionsRecycled.asTarget(),
+            signals.backend.backendConnectionsReused.asTarget(),
+            signals.backend.backendConnectionsBusy.asTarget(),
+            signals.backend.backendConnectionsUnhealthy.asTarget(),
+          ],
+          description='Rate of recycled, reused, busy, unhealthy, and accepted backend connections by Varnish Cache.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('conn/s')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withFillOpacity(20)
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls(false)
+        + g.panel.timeSeries.fieldConfig.defaults.custom.stacking.withMode('normal'),
+
+      // Sessions timeseries
+      sessionsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Sessions',
+          targets=[
+            signals.sessions.sessionsConnected.asTarget(),
+            signals.sessions.sessionsQueued.asTarget(),
+            signals.sessions.sessionsDropped.asTarget(),
```

Comment:
Same thing on usage of `irate` here which could be useful but feel as though the `increase` value may be more useful?